docs/pages/api.rst

```diff
@@ -1493,13 +1532,23 @@ Signature: ``Collection::reverse();``
 .. code-block:: php
 
     $collection = Collection::fromIterable(['a', 'b', 'c'])
-        ->reverse();
+        ->reverse(); // ['c', 'b', 'a']
 
 rsample
 ~~~~~~~
 
 Work in progress... sorry.
 
+Interface: `RSampleable`_
+
+Signature: ``Collection::rsample();``
+
+.. code-block:: php
+
+    $collection = Collection::fromIterable(range(1, 5));
+    $collection->rsample(1.0); // [1, 2, 3, 4, 5]
+    $collection->rsample(0.5); // will get about half of the elements at random
```

Comment:
I saw the description of `rsample` only says "work in progress" currently. What is this method based on? Is there anything we can link to? Is there a specific use case for it?

I tried to understand how it works and it looks to me like it returns a random set of elements based on the probability you supply: the higher the probability the more likely you will get close to all elements in the collection.

Reply:
This is it exactly. Nothing more, nothing less.

Reply:
Alright nice, I will add that in